src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -229,6 +229,40 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
                 }
             },
 
+            outerHTML: {
+                condition: domUtils.isElementNode,
+                get:       el => cleanUpHtml(el.outerHTML, el.parentNode && el.parentNode.tagName),
```

Comment:
It doesn't work. In ASPxUpload control uses `XMLDocument` instance with content like 
```xml
<status progress="100" totalSize="4" totalUploadedSize="4" contentType="" fileProgress="100" fileUploadedSize="4" fileSize="4" fileName="test.txt"></status>
```

It checks for  `xml.documentElement.outerHTML` being `undefined` to detect, it is valid XML document or not. In case of invalid XML, `documentElement` doesn't have `outerHTML` property, but it's `nodeType` is `1`, so it passes our `domUtils.isElementNode` check.